src/protocols/secure_channel/CASESession.cpp

```diff
@@ -165,6 +167,7 @@ CHIP_ERROR CASESession::Deserialize(CASESessionSerialized & input)
 
 CHIP_ERROR CASESession::ToSerializable(CASESessionSerializable & serializable)
 {
+    // TODO: IPK should not be serialized/deserialized. Remove it from the stored session state.
```

Comment:
Removing it will cause breakage, which will require version bump.

Reply:
That's true. Let me see if I can fix the unit test in this PR itself.